docs/_static/js/authcodescripts.js

```diff
@@ -0,0 +1,30 @@
+function onloadoauthcode() {
```

Comment:
Nit: We'll need a license header here. pandas-gbq inherited a BSD license, so it'll look a bit different from the Apache 2.0 header.

```suggestion


// Copyright (c) 2023 pandas-gbq Authors All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.


function onloadoauthcode() {
```